### PR TITLE
Add permissions to site

### PR DESCRIPTION
**Pull request by BarryShehadeh**

### CODE REVIEW

**Review by JeremyKelley — CHANGES_REQUESTED** [review]
Not sure if there's a Vroom -> Graph discrepancy but at least on the direct endpoint there are two changes needed.  In the POST method grantedTo should be grantedToIdentities and it should take an array rather than a single object.

**Review by JeremyKelley — APPROVED** [review]

File: api-reference/beta/api/site-delete-permission.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+author: "BarrySh"
+localization_priority: Normal
+ms.prod: "sharepoint"
+doc_type: apiPageType
+---
+
+# Delete permission
+Namespace: microsoft.graph
+
+Delete a permission object on a site.
```
Comment (Lauragra) [inline api-reference/beta/api/site-delete-permission.md:13]: Global: In the summary statements where you mention permission object, please also link to the permission resource topic.

File: changelog/Microsoft.FileServices.json [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **permissions** relationship to the [site](https://docs.microsoft.com/en-us/graph/api/resources/site?view=graph-rest-1.0) resource.",
+          "Target": "site"
+        }
+      ],
+      "Id": "aa7c9c04-2a12-458f-9bd5-a46c3b2fccd4",
+      "Cloud": "prd",
+      "Version": "v1.0",
+      "CreatedDateTime": "2021-02-05T01:49:05.616Z",
+      "WorkloadArea": "Files",
```
Comment (Lauragra) [inline changelog/Microsoft.FileServices.json:18]: The WorkloadArea here should be "Sites and lists" - confirm?

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please add TOC entry points for the new operation topics.

File: changelog/Microsoft.FileServices.json (hunk 2 above)
Comment (BarryShehadeh) [reply changelog/Microsoft.FileServices.json]: Correct.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**